localization/english/triggers/wc_character_triggers_l_english.yml

```diff
@@ -14,4 +14,9 @@
  
  AGE_IS_LESS_OR_EQUAL:0 "Is $NUM|V0$ years old or under"
  MY_AGE_IS_LESS_OR_EQUAL:0 "Your are $NUM|V0$ years old or under"
- THEIR_AGE_LESS_OR_EQUAL:0 "[CHARACTER.GetShortUIName|U] is $NUM|V0$ years old or under"
\ No newline at end of file
+ THEIR_AGE_LESS_OR_EQUAL:0 "[CHARACTER.GetShortUIName|U] is $NUM|V0$ years old or under"
+
+ stasis_trap_already_stunned_first_not:0 "I am #bold not#! stunned"
+ stasis_trap_already_stunned_first:0 "I am stunned"
+ stasis_trap_already_stunned_third_not:0 "[CHARACTER.GetShortUIName] is #bold not#! stunned"
+ stasis_trap_already_stunned_third:0 "[CHARACTER.GetShortUIName] is already stunned"
```

Comment:
Stunned should be a game concept here.
It's not clear why you can't use the decision.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117260306-fcb86080-ae5f-11eb-8262-663272520caa.png)

</details>